source/Halibut.Tests/ServiceModel/PendingRequestQueueFixture.cs

```diff
@@ -132,6 +131,44 @@ public async Task QueueAndWait_WhenRequestIsDequeued_ButPollingRequestQueueTimeo
             next.Should().BeNull();
         }
         
+        [Test]
+        public async Task QueueAndWait_WhenRequestIsDequeued_WithRelyOnConnectionTimeoutsInsteadOfPollingRequestMaximumMessageProcessingTimeout_WillWaitForeverUntilResponseIsSet()
+        {
+            // Arrange
+            const string endpoint = "poll://endpoint001";
+
+            var sut = new PendingRequestQueueBuilder()
+                .WithEndpoint(endpoint)
+                .WithPollingQueueWaitTimeout(TimeSpan.Zero) // Remove delay, otherwise we wait the full 20 seconds for DequeueAsync at the end of the test
```

Comment:
Maybe set this to `1s` otherwise we might be flooding the network with null requests and we might not be giving enough time to actually collect from the queue.

Reply:
We do this throughout this test file.

We should do them as a set. Do it as part of this PR?

Reply:
Oh i didn't realise this is how all other tests were configured, I think it makes sense to just do what is working :D